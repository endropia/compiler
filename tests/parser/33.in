var
    i: integer;
begin
    for i := 1 to 10 do
        writeln('*');

    for i := 1 to 10 do
    begin
            writeln('*');
             writeln('*');
    end;
end.