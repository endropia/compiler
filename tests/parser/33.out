program : Unnamed program
   var: 
      i
      type: integer
   stmts:
      for
         i
         to
            1
            10
               call
                  writeln
                     *
      for
         i
         to
            1
            10
               stmts:
                  call
                     writeln
                        *
                  call
                     writeln
                        *
